server/models/Game.js

```diff
@@ -30,13 +35,29 @@ const gameSchema = new mongoose.Schema({
     // }]
```

Comment:
The categories thing is breaking my display...we'll have to talk about how to fix this or populate my mongoose properly soon.

Reply:
Ok.  I think I ran into some issues too but then resolved it.  Not sure what the issue might be.